resources/scripts/Global.lua

```diff
@@ -79,3 +79,8 @@ registerCommand("setspeed", "commands/debug/SetSpeed.lua")
 registerCommand("nudge", "commands/debug/Nudge.lua")
 registerCommand("festival", "commands/debug/Festival.lua")
 registerCommand("permtest", "commands/debug/PermissionTest.lua")
+registerCommand("unlockaction", "commands/debug/UnlockAction.lua")
+registerCommand("wireframe", "commands/debug/ToggleWireframe.lua")
+registerCommand("invis", "commands/debug/ToggleInvisibility.lua")
+registerCommand("unlockaetheryte", "commands/debug/UnlockAetheryte.lua")
+registerCommand("teri", "commands/debug/ChangeTerritory.lua")
```

Comment:
I would like to use the existing `//gm` command structure instead of new debug commands, but this fine until I get off my butt and implement support for that in the Lua API